src/scripts/components/TimeEntriesList.tsx

```diff
@@ -35,11 +35,23 @@ const getTimeEntryDayGroups = (timeEntries: Array<Array<Toggl.TimeEntry>>): {[da
     }, {})
 };
 
+const TimeEntriesFooter = () => (
+  <Footer>
+    <a target="_blank" href="https://toggl.com/app/timer?utm_source=toggl-button&utm_medium=referral">
+      <Button>
+        See more on <strong>toggl.com</strong>
+      </Button>
+    </a>
+  </Footer>);
+
 interface TimeEntriesListProps {
   timeEntries: Toggl.TimeEntry[][];
   projects: IdMap<Toggl.Project>;
 };
-export default function TimeEntriesList (props: TimeEntriesListProps) {
+export default function TimeEntriesList (props?: TimeEntriesListProps) {
```

Comment:
props will never be `undefined`, so if you want to mark things as optional mark the keys in L48 and L49

Reply:
reverted :raised_hands: